AjuntamentdeBarcelona/decidim.barcelona
# Filters usability upgrades

# What and why

I have improved filters usability a bit doing two things:
- If a filter group is exclusive it renders as a collection of radio buttons
- If there is a filter active a clean filters link appears at the bottom to clean active filters
# QA

Visit either meetings or proposals page and set some filters. The clean filters link should appear and work properly.
# GIF Tax

![](https://media.giphy.com/media/Jj1Z1kUoqhi6I/giphy.gif)
